lib/navigation.js

```diff
@@ -30,23 +30,35 @@ module.exports = function () {
             let frontmatter = files[url + '/index.html']
             // if we have frontmatter for that file, create subitem
             if (frontmatter) {
-              if (frontmatter.status && frontmatter.status === 'Draft') {
-                // If the page is labelled as a draft, do not show in the navigation.
-                continue
-              }
-
               let subitem = {
                 'url': url,
                 'label': frontmatter.title
               }
               // if frontmatter contains `theme` data, attach it to navigation item for grouping
-              if (frontmatter.theme) {
+              if (frontmatter.theme !== undefined) {
                 subitem.theme = frontmatter.theme
               }
+              // if frontmatter contains 'order' data, use it for ordering the navigation
+              if (frontmatter.order !== undefined) {
+                subitem.order = frontmatter.order
+              }
               // add subitem to navigation
               navigation[item].items.push(subitem)
             }
           }
+
+          // if 'order' is set, use it to sort the navigation
+          if (navigation[item].items[0].order !== undefined) {
+            navigation[item].items.sort((a, b) => {
```

Comment:
I believe this can just be `items.sort((a, b) => a.order - b.order)`

> To compare numbers instead of strings, the compare function can simply subtract b from a.
> 
> – https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort